src/pages/index.tsx

```diff
@@ -26,8 +26,8 @@ const HomePage = () => (
     <Hero />
     <Features />
     <Grid>
-      <Box as="hr" marginBottom={0} />
-      <Box paddingVertical={80} paddingVerticalMd={120}>
+      <Box as="hr" margin={0} />
+      <Box paddingVertical={64} paddingVerticalMd={120}>
```

Comment:
It appears that the bigger padding (120px) should kick in on the bigger screens. Could you set it as `paddingVerticallg={120}` instead? See how the padding decreases in the features' section on responsive changes. Thanks!